lib/cocina_checker.rb

```diff
@@ -18,21 +18,21 @@ def initialize(repos:, tag: nil)
   end
 
   def check_cocina
-    puts "repos to Cocina check: #{repos.map(&:name).join(', ')}"
+    render_markdown('***')
+    render_markdown("# Checking Cocina version (#{tag || 'default branch'})")
 
     unique_values = group_by_unique_values(version_map)
-    puts '------- COCINA REPORT -------'
-    if tag
-      puts "  for tag #{tag} of repos"
-    else
-      puts '  for default branches of repos'
-    end
-    puts 'Found these versions of cocina in use:'
     unique_values.sort.each do |version, repos|
-      puts "\t#{version}"
-      repos.sort.each do |repo|
-        puts "\t\t#{repo}"
-      end
+      report_table << [
+        version,
+        repos.sort.join("\n")
+      ]
+    end
+
+    if unique_values.empty?
+      render_markdown('## No repositories depend on Cocina')
```

Comment:
Add a new informational message when deploying no repos that have Cocina in them.